packages/web-components/@sfx/sayt/stories/index.ts

```diff
@@ -25,6 +25,8 @@ const productsDataReceivedEvent = new CustomEvent('sfx::provide_products',
           promo: '25% off',
           imageSrc: 'https://images.unsplash.com/photo-1542291026-7eec264c27ff?ixlib=rb-1.2.1&auto=format&fit=crop&h=350&q=80',
           imageAlt: 'A spicy red shoe',
+          info: 'Info 1',
+          info2: 'Info 2',
```

Comment:
These don't appear to be used anywhere in the codebase.

Reply:
They were added just to show how other random pieces of information added to the story and tiles stretching to a uniform size. Can be removed if you would rather.

Reply:
Ah, I see. SAYT currently doesn't display this information because it doesn't display products right now. We can leave it in. It wouldn't be too bad to have a story (or use an existing one) in the `Products` section that displays this functionality, but I realize that's out of scope.